src/main/java/edu/ucsd/sbrg/bigg/ModelPolisher.java

```diff
@@ -284,6 +289,7 @@ private void initParameters(SBProperties args) {
       fObj = fObjectives.substring(1, fObjectives.length() - 1).split(":");
     }
     parameters.annotateWithBiGG = args.getBooleanProperty(ModelPolisherOptions.ANNOTATE_WITH_BIGG);
+    parameters.outputCOMBINE = args.getBooleanProperty(ModelPolisherOptions.OUTPUT_COMBINE);
```

Comment:
Like mentioned in the README comment. All these parameters should be documented with what they do and what values they can take and what their defaults are.